tools/accuracy_checker/accuracy_checker/main.py

```diff
@@ -36,254 +36,322 @@
 }
 
 
-def build_arguments_parser():
-    parser = ArgumentParser(description='Deep Learning accuracy validation framework', allow_abbrev=False)
-    parser.add_argument(
+def add_common_args(parser):
+    common_args = parser.add_argument_group('Common arguments')
+    common_args.add_argument(
         '-d', '--definitions',
         help='path to the yml file with definitions',
         type=get_path,
         required=False
     )
-    parser.add_argument(
+    common_args.add_argument(
         '-c', '--config',
         help='path to the yml file with local configuration',
         type=get_path,
         required=True
     )
-    parser.add_argument(
+    common_args.add_argument(
         '-m', '--models',
         help='prefix path to the models and weights',
         type=partial(get_path, is_directory=True),
         required=False,
         nargs='+'
     )
-    parser.add_argument(
+    common_args.add_argument(
         '-s', '--source',
         help='prefix path to the data source',
         type=partial(get_path, is_directory=True),
         required=False
     )
-    parser.add_argument(
+    common_args.add_argument(
         '-a', '--annotations',
         help='prefix path to the converted annotations and datasets meta data',
         type=partial(get_path, is_directory=True),
         required=False
     )
-    parser.add_argument(
-        '-e', '--extensions',
-        help='prefix path to extensions folder',
-        type=partial(get_path, check_exists=False),
-        default=Path.cwd(),
+    common_args.add_argument(
```

Comment:
please move logging file to tool settings arguments

Reply:
Moved